src/pages/patientView/mutationalSignatures/MutationalSignatureBarChart.tsx

```diff
@@ -53,25 +51,25 @@ theme.legend.style.data = {
     stroke: 'black',
 };
 
-export function formatLegendTopAxisPoints(data: IMutationalCounts[]) {
+/*export function formatLegendTopAxisPoints(data: IMutationalCounts[]) {
     const constant_value = _.max(data.map(item => item.value)) || 0;
     const groupedData = _.groupBy(getColorsForSignatures(data), g => g.group);
     return getColorsForSignatures(data).map(entry => ({
         x: entry.mutationalSignatureLabel,
         y: constant_value + constant_value * 0.1,
         color: entry.colorValue,
     }));
-}
+}*/
 
-export function transformMutationalSignatureData(dataset: IMutationalCounts[]) {
+/*export function transformMutationalSignatureData(dataset: IMutationalCounts[]) {
     const transformedDataSet = dataset.map((obj: IMutationalCounts) => {
         const transformedDataSet = dataset.map((obj: IMutationalCounts) => {
             let referenceTransformed = -Math.abs(obj.value);
             return { ...obj, referenceTransformed };
         });
         return transformedDataSet;
     });
-}
+}*/
```

Comment:
what were these used for previously?

Reply:
`formatLegendTopAxisPoints` is not needed anymore since this behavior is captured in the `@computed formatLegendTopAxisPoints`. The second function `transformMutationalSignatureData` is to show the reference counts but we will not show them in this chart. (I have a separate branch with the reference signature so the function is not lost)